src/router/index.js

```diff
@@ -0,0 +1,24 @@
+const Router = require("express").Router;
+const userController = require("../controllers/user-controller");
+const {
+  registrationValidator,
+  loginValidator,
+  validate,
+} = require("../validator/validator");
+
+const router = new Router();
+
+router.post(
+  "/registration",
+  registrationValidator,
+  validate,
+  userController.registration
+);
+
+router.post("/login", loginValidator, validate, userController.login);
+
+router.post("/logout", userController.logout);
+
+router.get("/refresh", userController.refresh);
+
```

Comment:
why is the user routes file called index.js?
index.js - you will have to structure all other files with routes. I wrote in the general chat how to structure the routes correctly